### PR TITLE
Create ISorokos.json

### LINKED ISSUE #8812: GitHub flow
Hello! :wave: Please add me as a collaborator to this repo.

### PR DESCRIPTION
**Pull request by ISorokos**
Committing user json w/ longitude+latitude info
Did I get this right @githubteacher ?
closes #8812

### CODE REVIEW

**Comment by ISorokos** [comment]
Hm, what gives ?
Why did Travis fail ?

File: _pins/ISorokos.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: ISorokos
+latitude: 53.7167
+longitude: -0.3333
```
Comment (GregJablonski) [inline _pins/ISorokos.json]: Looks Good 👍

**Comment by GregJablonski** [comment]
🥇

**Comment by guillaume-ro-fr** [comment]
@ISorokos Please rebase your branch from master branch (Travis test fixed)

**Pull request closed without merging**